telehealth-example/src/main/java/com/pubnub/components/example/getting_started/ui/view/Chat.kt

```diff
@@ -0,0 +1,203 @@
+import com.pubnub.components.chat.ui.component.common.ThemeDefaults
+import com.pubnub.components.chat.ui.component.input.MessageInput
+import com.pubnub.components.chat.ui.component.input.TypingIndicatorContent
+import com.pubnub.components.chat.ui.component.menu.Copy
+import com.pubnub.components.chat.ui.component.menu.React
+import com.pubnub.components.chat.ui.component.message.LocalMessageListTheme
+import com.pubnub.components.chat.ui.component.message.MessageList
+import com.pubnub.components.chat.ui.component.message.MessageListTheme
+import com.pubnub.components.chat.ui.component.message.MessageUi
+import com.pubnub.components.chat.ui.component.presence.Presence
+import com.pubnub.components.chat.ui.component.provider.LocalChannel
+import com.pubnub.components.chat.viewmodel.message.MessageViewModel
+import com.pubnub.components.chat.viewmodel.message.MessageViewModel.Companion.defaultWithMediator
+import com.pubnub.components.chat.viewmodel.message.ReactionViewModel
+import com.pubnub.components.example.getting_started.R
+import com.pubnub.components.example.getting_started.ui.theme.MessageBackgroundColor
+import com.pubnub.components.example.getting_started.ui.theme.MessageOwnBackgroundColor
+import com.pubnub.components.example.getting_started.ui.theme.Shapes
+import com.pubnub.framework.data.ChannelId
+import kotlinx.coroutines.flow.Flow
+
+object Chat {
+
+    @Composable
+    internal fun Content(
+        messages: Flow<PagingData<MessageUi>>,
+        presence: Presence? = null,
+        onMessageSelected: (MessageUi.Data) -> Unit,
+        onReactionSelected: ((React) -> Unit)? = null,
+        patientUUID: String,
+        patientName: String
+    ) {
+
+        val customTheme = ThemeDefaults.messageList(
+            messageOwn = ThemeDefaults.message(
+                text = text(
+                    Modifier
+                        .background(color = MessageOwnBackgroundColor)
+                        .padding(2.dp)
+
+                ),
+                shape = ShapeThemeDefaults.shape(shape = Shapes.medium),
+                modifier = Modifier
+            ),
+            message = ThemeDefaults.message(
+                text = text(
+                    modifier = Modifier
+                        .background(color = MessageBackgroundColor)
+                        .padding(2.dp)
+                ),
+                shape = ShapeThemeDefaults.shape(shape = Shapes.medium),
+                modifier = Modifier.clip(RoundedCornerShape(10.dp))
+            )
+        )
+        val localFocusManager = LocalFocusManager.current
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .pointerInput(Unit) {
+                    detectTapGestures(onTap = {
+                        localFocusManager.clearFocus()
+                    })
+                }
+        ) {
+            MessageListTheme(customTheme) {
+                Row(
```

Comment:
Should it be centered or something? Looks different on various devices:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/7523141/197480117-7c528ecb-1828-4eed-a963-94a38a327d18.png">

Reply:
Done